### PR TITLE
docs: Update url in kata vra document

### LINKED ISSUE #8065: docs: update ur in kata vra document

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the url in kata vra document.

Fixes #8065

### CODE REVIEW

File: docs/design/kata-vra.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ and perform DMA transactions _anywhere_.
 
 The second feature is ACS (Access Control Services), which controls which
 devices are allowed to communicate with one another and thus avoids improper
-routing of packets irrespectively of whether IOMMU is enabled or not.
+routing of packets `irrespectively` of whether IOMMU is enabled or not.
```
Comment (cmaf) [inline docs/design/kata-vra.md:46]: @GabyCT is this because of the spell check?

Comment (GabyCT) [reply docs/design/kata-vra.md:46]: @cmaf exactly kata spell check sends those warnings which makes `make static checks` fail, before on the tests repository we had a file to add words and so on but as that repo soon will be disabled we need to do this

**Review by zvonkok — APPROVED** [review]
LGTM, thanks for updating.

**Review by dborquez — APPROVED** [review]

**Pull request merged**